modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -323,12 +323,12 @@ def detect_current_status(data)
             return 'NO_STATUS_PROVIDED'
           end
 
-          phase_data = if data[:phase_type].present?
+          phase_data = if data[:claim_status] == 'CAN'
```

Comment:
modified to only pull specific for cancelled at cancelled have COMPLETE as latest phase type